include/mbedtls/ssl.h

```diff
@@ -2720,17 +2713,29 @@ int mbedtls_ssl_conf_own_cert( mbedtls_ssl_config *conf,
  *                 hence need not be preserved by the caller for the lifetime
  *                 of the SSL configuration.
  *
+ * \note           While this function may be called multiple times to
+ *                 register multiple PSKs, the number of supported PSKs
+ *                 is implementation-defined. Once the limit is reached,
+ *                 this function fails, maintaining the PSKs previously
+ *                 configured and ignoring the excess request.
+ *                 This behavior is in contrast to Mbed TLS 2.x, where
+ *                 later invocations would overwrite the effect of earlier
+ *                 calls.
```

Comment:
Note for the future (no change required in this PR): when we add support for multiple PSKs, we'll want to document clearly what happens when a user attempts to provision two PSKs with the same identity. (My first thought is it should be an error.)

Reply:
Good point. I think this should be specified already now, though.

Reply:
On second thought, I agree that we can leave this for when we actually support more than one PSK.